### PR TITLE
api: document changed behavior of the `Aliases` field in v1.45

### PR DESCRIPTION
**Pull request by neersighted**
**- What I did**

Address my concerns with the deprecations in https://github.com/docker/cli/pull/4949#pullrequestreview-1948948034

### CODE REVIEW

File: docs/api/version-history.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ keywords: "API, Docker, rcli, REST, documentation"
 * `GET /images/{name}/json` no longer includes the `Container` and
   `ContainerConfig` fields. To access image configuration, use `Config` field
   instead.
+* The `Aliases` field returned in calls to `GET /containers/{name:.*}/json` will no longer
```
Comment (thaJeztah) [inline docs/api/version-history.md:28]: can you make this present tense? (no longer contains)?

**Review by vvoland — APPROVED** [review]

File: docs/api/version-history.md [hunk 2]
```diff
@@ -25,6 +25,11 @@ keywords: "API, Docker, rcli, REST, documentation"
 * `GET /images/{name}/json` no longer includes the `Container` and
   `ContainerConfig` fields. To access image configuration, use `Config` field
   instead.
+* The `Aliases` field returned in calls to `GET /containers/{name:.*}/json` no
+  longer contains contain the short container ID, but instead will reflect
```
Comment (thaJeztah) [inline docs/api/version-history.md:29]:
```suggestion
  longer contains the short container ID, but instead will reflect
```

**Review by robmry — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
@vvoland I think this one is fine to merge without waiting _all of CI_ to finish (to your discretion)

**Review by akerouanton — COMMENTED** [review]
LGTM. Thanks!

**Pull request merged**